addon/components/tether-tooltip-and-popover.js

```diff
@@ -27,6 +27,16 @@ function cleanNumber(stringOrNumber) {
 
 export default EmberTetherComponent.extend({
 
+  passedPropertiesObject: null,
+  setPropertiesWithPassedPropertiesObject: on('didReceiveAttrs', function() {
+    this._super(...arguments);
+
+    let passedPropertiesObject = this.get('passedPropertiesObject');
+    if (passedPropertiesObject) {
+      this.setProperties(passedPropertiesObject);
```

Comment:
I'm not sure if this is what we want. You're just setting the properties, but not creating a binding. 

let's say we have prop1 as a passedProperty. Then if you change prop1 on the context, it will flow to the target, but it won't flow from the target to the `tether-tooltip-and-popover` since didReceiveAttrs won't be called again, so the actual component will be left with the old value. 

Can you test it? This may not be an issue, but if I'm right, it will be a bad way to break... 

In case it's an issue, it may be as simple as setting adding the same props with a `Binding` suffix or explicitly create bindings between wrapper and wrappee for each property on passedPropertiesObject. 

Also, I'm wondering if it's better to generalize this use case and apply it here, it seems like the "transparent wrapper" or "transparent proxy" has value on its own, but there are also a lot of edge cases that might be easier to test in isolation. I started doing something like this for https://github.com/zenefits/ember-component-apply and something similar and for https://github.com/toranb/ember-cli-hot-loader/pull/25. While both of those approaches work as a "transparent proxy" for components, I don't think they're applicable for ember-tooltips since we rely in one case on transpiling templates on the fly and in the other one to rewrite the AST at buildtime. I only mention those two, because they already cover some of the edge cases for "transparent proxies" and you can look at some of the tests cases for them.